### ISSUE TITLE
Filter for capability for Optin Monster

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/semperpluginstheme#3]
In the functions.php file of the current child theme is this filter:

add_filter( 'optin_monster_api_menu_cap', 'om_allow_editors_in_menu' );

I believe this was added so that Tony could edit optins.  We need to check if this is still required and add this code to the new theme if it is.

**Comment by michaeltorbert** [comment]
I don't think we need this anymore. If it's still in there, it can be removed.
